index.html

```diff
@@ -70,6 +70,215 @@ <h1>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <div class="card-button">
+              <button type="button" class="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="card-container">
+          <div class="card-image"></div>
+          <div class="card-body">
+            <div class="card-title">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <div class="card-content">
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <div class="card-button">
+              <button type="button" class="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="card-container">
+          <div class="card-image"></div>
+          <div class="card-body">
+            <div class="card-title">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <div class="card-content">
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <div class="card-button">
+              <button type="button" class="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="card-container">
+          <div class="card-image"></div>
+          <div class="card-body">
+            <div class="card-title">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <div class="card-content">
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <div class="card-button">
+              <button type="button" class="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="about-myself">
+      <img src="images/frameup.png" alt="" class="img-up">
+      <h2 class="title">
+        About me
+      </h2>
+      <p class="more">
+        I can help you buils a product, feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don't
+        hesitate to contact me.
+      </p>
+      <img src="images/Framed.png" alt="" class="img-down">
+      <button type="button" class="button">
+        Get My Resune
+      </button>
+    </section>
+    <section id="experience">
+      <div class="card-exp">
```

Comment:
- [ ] Please make your **experience** in `ul` & all **card-exp** in `li` tags instead of making them in `section` and `div` tags. Because these cards are list items and for every list, you should use `ul` and `li` tags. :blush: